src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp

```diff
@@ -1084,9 +1089,8 @@ FixedwingPositionControl::control_auto_position(const float control_interval, co
 
 	float tecs_fw_thr_min;
 	float tecs_fw_thr_max;
-	float tecs_fw_mission_throttle;
 
-	float mission_throttle = _param_fw_thr_cruise.get();
+	float mission_throttle = _param_fw_thr_trim.get();
```

Comment:
it looks like `mission_throttle` is not used anywhere except to be set to `cruising_throttle` and then checked against `fw_thr_min`. maybe better to just have this:
```cpp
if (mission_throttle < _param_fw_thr_min.get()) {
        /* enable gliding with this waypoint */
        _tecs.set_speed_weight(2.0f);
        tecs_fw_thr_min = 0.0;
        tecs_fw_thr_max = 0.0;
} else {
        tecs_fw_thr_min = _param_fw_thr_min.get();
        tecs_fw_thr_max = _param_fw_thr_max.get();
}
```
handled inside the cruising throttle check?